index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Overnight Oats</h1>
+    <nav>
+      <a href="Ingredients.html">Ingredients</a>
```

Comment:
If you replace this with a list of links, I can show you how to update just this page content without having to switch between HTML pages ([this demo](https://a11y-tabs.glitch.me/) shows very basic tabs -- notice how the URL doesn't change -- and I can help you use this in your project).

So, inside your `nav`, you'd want to replace your current link with:

```suggestion
      <ul>
          <li> <a href="#directions">Directions</a></li>
          <li><a href="#ingredients">Ingredients</a></li>
      </ul>
```

Great use of a `nav` element here, btw. 👍🏻